e2e-tests/fixtures/companies.js

```diff
@@ -0,0 +1,42 @@
+import {
+  COMPANIES_HOUSE_NUMBER, COMPANIES_HOUSE_NUMBER_NO_SIC_CODE, COMPANIES_HOUSE_NUMBER_MULTIPLE_SIC_CODES,
+} from '../constants/examples';
+import { EXPORTER_BUSINESS as EXPORTER_BUSINESS_FIELD_IDS } from '../constants/field-ids/insurance/business';
+import mockApplication from './application';
+import mockSicCodes from './sic-codes';
+
+const {
+  COMPANY_HOUSE: { INDUSTRY_SECTOR_NAME },
+} = EXPORTER_BUSINESS_FIELD_IDS;
+
+const baseCompany = mockApplication.EXPORTER_COMPANY;
+
+const mockCompanies = {
```

Comment:
is there a company with no financial year end date?

Reply:
There is the example company numbers constants but not in this file currently.

Good shout, will add this in.